python/ray/serve/tests/test_gradio.py

```diff
@@ -0,0 +1,71 @@
+import os
+import sys
+import pytest
+import requests
+
+import ray
+from ray import serve
+from ray.serve.gradio import GradioServer
+import gradio as gr
+
+
+@pytest.fixture
+def serve_start_shutdown():
+    ray.init()
+    serve.start()
+    yield
+    serve.shutdown()
+    ray.shutdown()
+
+
+pytest.mark.skipif(sys.version_info < (3, 7), reason="Need Python 3.7.")
```

Comment:
```suggestion
@pytest.mark.skipif(sys.version_info < (3, 7), reason="Need Python 3.7.")
```

This is a decorator.